### PR TITLE
Upgrade to pChart2 - refs CT#7408

### PR DESCRIPTION
**Pull request by danbarretodev**
Upgrade from pChart 1.x to 2.x, using https://github.com/szymach/c-pchart
To generate autoload files use:
``` 
composer install

The pchart (old version) library was removed from library path

### CODE REVIEW

File: main/css/opensans/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (jmontoyaa) [inline main/css/opensans/LICENSE.txt]: The "opensans" font folder should be located here:

main/fonts/opensans

File: main/inc/lib/autoload.class.php [hunk 2]
```diff
@@ -751,7 +751,6 @@ static public function &map()
         $result['MultipleAnswerCombination'] = '/main/exercice/multiple_answer_combination.class.php';
```
Comment (jmontoyaa) [inline main/inc/lib/autoload.class.php]: Adding this will be not needed soon:

https://support.chamilo.org/issues/7501

File: composer.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
         "twig/twig": "1.*",
         "michelf/php-markdown": "1.4.1",
         "emojione/emojione": "1.3.0",
-        "zendframework/zend-config": "2.3.3"
+        "zendframework/zend-config": "2.3.3",
+        "szymach/c-pchart": "1.*"
```
Comment (jmontoyaa) [inline composer.json]: :+1:

**Comment by jmontoyaa** [comment]
After those changes this could be merge, please update from 1.10x there were some changes to use composer autoload instead the custom autoloader.

**Comment by danbarretodev** [comment]
Move opensans folder to /main/fonts.
Execute composer install to add pChart
Rebase commits onto 1.10.x

Wich changes do yo mean?

**Comment by jmontoyaa** [comment]
All of them.

**Comment by ywarnier** [comment]
Julio, could you review it again, please?

**Pull request merged**